### PR TITLE
v2.0.0+galaxy1

### PR DESCRIPTION
**Pull request by RJMW**

### CODE REVIEW

File: tools/dimspy/align_samples.xml [hunk 1]
```diff
@@ -29,6 +31,9 @@
         <param name="ppm" argument="--ppm" type="float" value="2.0" label="Ppm error tolerance" help="Maximum tolerated m/z deviation across samples in parts per million (ppm)." />
         <param name="delimiter" argument="--delimiter" type="hidden" value="tab" />
         <expand macro="hdf5_pm_to_txt" />
+        <section name="adv" title="Advanced options" expanded="False">
```
Comment (bgruening) [inline tools/dimspy/align_samples.xml:34]: a hidden section with one parameter, is just increasing complexity and does not hide much, isn't it?

File: tools/dimspy/align_samples.xml [hunk 2]
```diff
@@ -47,6 +52,22 @@
...
+            <conditional name="hdf5_to_txt" >
+                <param name="standard" value="True"/>
+                <param name="comprehensive" value="False"/>
+                <param name="samples_representations" value="rows"/>
+                <param name="matrix_attr" value="intensity"/>
+            </conditional>
+            <section name="adv">
+                <param name="block_size" value="10000"/>
+            </section>
+            <output name="hdf5_file_out" file="pm_as.h5" ftype="h5" compare="sim_size"/>
```
Comment (bgruening) [inline tools/dimspy/align_samples.xml:68]: we can now has_size: https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-tests-test-output-assert-contents or special hdf5 asserts like `has_h5_keys` to test for.

**Review by bgruening — COMMENTED** [review]
Very nice!

File: tools/dimspy/align_samples.xml (hunk 1 above)
Comment (RJMW) [reply tools/dimspy/align_samples.xml:34]: Noted - more advanced parameters will be added in the next version - I would like to keep this as a place holder.

File: tools/dimspy/align_samples.xml (hunk 2 above)
Comment (RJMW) [reply tools/dimspy/align_samples.xml:68]: Thanks for making me aware of these additional asserts features. Very useful!

**Pull request merged**